core/src/main/scala/cats/MonadTrans.scala

```diff
@@ -0,0 +1,17 @@
+package cats
+
+/**
+ * A type class which abstracts over the ability to lift an M[A] into a
+ * MonadTransformer
+ */
+trait MonadTrans[MT[_[_], _]] {
+
+  /**
+   * Lift a value of type M[A] into a monad transformer MT[M, A]
+   */
+  def liftT[M[_]: Monad, A](ma: M[A]): MT[M, A]
```

Comment:
scalaz [does it](https://github.com/scalaz/scalaz/blob/series/7.3.x/core/src/main/scala/scalaz/MonadTrans.scala#L16). Not sure if it's useful. But it makes sense in theory.